src/layouts/Layout.astro

```diff
@@ -246,29 +247,42 @@ function isPathActive(path) {
     {
       displaySponsors && (
         <section id="sponsors" class="bg-gray-800">
-          <div class="m-auto max-w-5xl px-4 py-16 flex flex-col sm:flex-row gap-20 sm:gap-0 items-center justify-around">
-            <div class="flex flex-col items-center sm:items-start">
+          <div class="m-auto max-w-5xl px-4 py-16 flex flex-col md:flex-row gap-20 md:gap-0 items-center justify-around">
+            <div class="flex flex-col items-center md:items-start">
               <span class="text-lg font-semibold tracking-widest text-white">
-                SPONSORS
+                PLATINUM SPONSORS
               </span>
               <a class="flex justify-center mt-4" href="https://fly.io/">
-                <Image src={flyioLogoWithText} alt="Flyio Logo" width="250" />
+                <Image src={flyioLogoWithText} alt="Flyio Logo" width="270" />
               </a>
             </div>
-            <div class="flex flex-col items-center sm:items-start">
-              <span class="text-base tracking-widest text-white">
-                SUPPORTERS
-              </span>
-              <a
-                class="flex justify-center mt-4"
-                href="https://huggingface.co/"
-              >
-                <Image
-                  src={hfLogoWithText}
-                  alt="Hugging Face Logo"
-                  width="180"
-                />
-              </a>
+            <div class="flex flex-col items-center md:items-start">
+              <span class="text-base tracking-widest text-white">SPONSORS</span>
+              <div class="flex flex-row space-x-8">
+                <div>
+                  <a
+                    class="flex justify-center mt-4"
+                    href="https://huggingface.co/"
+                  >
+                    <Image
+                      src={hfLogoWithText}
+                      alt="Hugging Face Logo"
+                      width="150"
+                    />
+                  </a>
+                </div>
+                <div style="width:100px">
```

Comment:
I had to add that `style="width:100px"` instead of setting the `width=100px` in the Image component as we did for Fly and HF because Tigris' image is PNG, not SVG.

When I added `width=100px` as an attribute to the `<Image /` Astro component for the Tigris file, the image was automatically resized to 100px width, which doesn't have enough resolution for high-res displays.

Astro [has a solution for](https://docs.astro.build/en/guides/images/#densities) that, but only since version 3.3.0, and we're using version 3.1.2.

If anyone thinks the current solution needs to be improved and has other ideas, I'm open to it. 😄